.github/workflows/test-build.yml

```diff
@@ -6,7 +6,7 @@ jobs:
      runs-on: [ubuntu-latest]
      strategy:
        matrix:
-         go_version: [1.13]
+         go_version: [1.19]
```

Comment:
Why drop the support to version before 1.19?

Reply:
Versions <=1.19 are still supported. Do you want to put them in the matrix?

Reply:
I was suggesting about testing against at least 1.18; since we're not using generics we could still test at least the last 3 versions (17,18,19)

Reply:
Okay, i'll update the matrix versions

Reply:
done